presto-native-execution/CMakeLists.txt

```diff
@@ -68,7 +68,6 @@ if(PRESTO_ENABLE_PARQUET)
   set(VELOX_ENABLE_PARQUET
       ON
       CACHE BOOL "Enable Parquet support")
-  add_definitions(-DPRESTO_ENABLE_PARQUET)
```

Comment:
Let's also remove this from L47 and 67.

Reply:
We need those lines to control the `VELOX_ENABLE_PARQUET` option.

Reply:
@mbasmanova I added some comments for the Presto options. Can you take another look?